templates/show_params.mako

```diff
@@ -271,6 +271,30 @@ ${ job.command_line | h }</pre>
     %endfor
 %endif
 
+%if trans.user_is_admin():
+<h3>Destination Parameters</h3>
+    <table class="tabletip">
+        <tbody>
+            <tr><th scope="row">Runner</th><td>${ job.job_runner_name }</td></tr>
+            <tr><th scope="row">Runner Job ID</th><td>${ job.job_runner_external_id }</td></tr>
+            <tr><th scope="row">Handler</th><td>${ job.handler }</td></tr>
+            %if job.destination_params:
+            %for (k, v) in job.destination_params.items():
+                <tr><th scope="row">${ k | h }</th>
+                    <td>
+                        %if str(k) in ('nativeSpecification', 'rank', 'requirements'):
+                        <pre style="white-space: pre-wrap; word-wrap: break-word;">${ v | h }</pre>
```

Comment:
@jmchilton the default-singularity container is reported but not the actually used Singularity container. Any idea why this is the case?

Reply:
Yeah - I don't think we actually record that unfortunately. We have a couple different issues tracking that deficit: https://github.com/galaxyproject/galaxy/issues/4191 (bullet 2.2) / #5134 (bullet 3.1.2).